src/vdaf/poplar1.rs

```diff
@@ -407,6 +412,8 @@ impl Poplar1AggregationParam {
     ///
     /// * The list of prefixes is empty.
     /// * The prefixes have different lengths (they must all be the same).
+    /// * The prefixes are longer than 2^16 bits.
+    /// * There are more than 2^32 - 1 prefixes.
     //
     // TODO spec: Ensure that prefixes don't repeat. To make this check easier, consider requiring
     // them to appear in alphabetical order.
```

Comment:
Based on the checks on L439-449, the spec must have been updated to do this. Can this comment be deleted?